### DISCUSSION TITLE
apply custom theme on a custom page

### DISCUSSION THREAD

**Discussion by amirmmdz** [filamentphp/filament#13137] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hi how can i apply a custom theme on a custom page?
my page blade path is : resources/views/filament/pages/reset-password.blade
my page class path is : App\Filament\Pages\Auth\ResetPasswordPage
and my custom theme css file => resources/css/filament/admin/theme.css

**Comment by leandrocfe** [comment]
It should work if you add `'./resources/views/filament/**/*.blade.php',` in the content (tailwind.config.js)
Did you run `npm run build`?
